### DISCUSSION TITLE
Head Feature

### DISCUSSION THREAD

**Discussion by AlmostPanda** [BG-Software-LLC/SuperiorSkyblock2#1847] [Pending Feature Requests]
Hi, I would like to request the addition of option to set the skull skin using placeholder different then %superior_player_texture%
Something like:
skull: head-%player-name% or another possible placeholder which would include player name

PS: I want to create something like Leaderboard using SSB custom menus but I cant get different skin then mine
If u know any other way how to do it please let me know

**Comment by Davisiiiik** [comment]
Just a suggestion, this could be done using PlaceholderAPI ParseOther extension:
[https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#parseother](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Placeholders#parseother)

**Comment by OmerBenGera** [comment]
The `/is top` menu already has such feature, and I can't think about any other menu that has a leaderboard design.
